### PR TITLE
lastCommit

### PR DESCRIPTION
**Pull request by DenysStetsenko54321**

### CODE REVIEW

File: src/pages/CheckoutInfo.page.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    async fillInfo(firstName, lastName, postalCode) {
+        await this.firstName.fill(firstName);
+        await this.lastName.fill(lastName);
+        await this.postalCode.fill(postalCode);
+    }
+
+    async clickContinue() {
+        await this.continueButton.click();
+        return new CheckoutOverviewPage(this.page);
```
Comment (AlinaZhykharieva) [inline src/pages/CheckoutInfo.page.js]: It is a bit confusing to have such a method that does something with not this page

File: src/pages/CheckoutComplete.page.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    get thankYouMessage() { return this.page.locator('h2.complete-header'); }
+
+    get dispatchMessage() { return this.page.locator('div.complete-text'); }
+
+    get backHomeButton() { return this.page.locator('#back-to-products'); }
+
+    async verifyThankYouMessage() {
+        await expect(this.thankYouMessage).toBeVisible();
+        await expect(this.thankYouMessage).toHaveText('Thank you for your order!');
```
Comment (AlinaZhykharieva) [inline src/pages/CheckoutComplete.page.js]: better not to have expects in the page object
and text that we are checking now on a really low level in the test automation framework

Comment (AlinaZhykharieva) [reply src/pages/CheckoutComplete.page.js]: better to see that checks in the spec file

File: src/pages/CheckoutComplete.page.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    }
+
+    async verifyDispatchMessage() {
+        await expect(this.dispatchMessage).toBeVisible();
+        await expect(this.dispatchMessage).toHaveText('Your order has been dispatched, and will arrive just as fast as the pony can get there!');
```
Comment (AlinaZhykharieva) [inline src/pages/CheckoutComplete.page.js]: the same as in previous comment

**Review by ovartem — APPROVED** [review]
Well done!
I'd not add `expect` functions to page objects but this is not that critical so I approve this PR.
Really great job! Is it chat GPT generated code or ? :)

**Comment by DenysStetsenko54321** [comment]
@ovartem it's chat GPT)

**Pull request merged**